chromium_src/chrome/browser/ui/views/tabs/tab_strip.cc

```diff
@@ -15,8 +16,35 @@
 #define AddTab(TAB, MODEL_INDEX, PINNED) \
   AddTab(std::make_unique<BraveTab>(this), MODEL_INDEX, PINNED)
 #define TabHoverCardController BraveTabHoverCardController
+#define TabContainer BraveTabContainer
+#define TAB_DRAG_CONTEXT_CALCULATE_INSERTION_INDEX_FOR_VERTICAL_TABS           \
+  if (tabs::features::ShouldShowVerticalTabs()) {                              \
+    const int ideal_y =                                                        \
+        candidate_index == 0 ? 0 : ideal_bounds(candidate_index - 1).bottom(); \
+    const int distance = std::abs(dragged_bounds.y() - ideal_y);               \
+    if (distance < min_distance) {                                             \
+      min_distance = distance;                                                 \
+      min_distance_index = candidate_index;                                    \
+    }                                                                          \
+    continue;                                                                  \
+  }
+
+#define TAB_DRAG_CONTEXT_CALCULATE_BOUNDS_FOR_DRAGGED_VIEWS_VERTICAL_TABS \
+  if (tabs::features::ShouldShowVerticalTabs()) {                         \
+    std::vector<gfx::Rect> bounds;                                        \
+    int y = 0;                                                            \
+    for (const TabSlotView* view : views) {                               \
+      const int height = view->height();                                  \
+      bounds.push_back(gfx::Rect(0, y, view->width(), view->height()));   \
```

Comment:
`push_back` -> `emplace_back` and `view->height()` -> `height`

Reply:
Cool! Done.